documentation/modules/cruise-control/proc-implementing-rebalance-proposal.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// assembly-cruise-control-concepts.adoc
+
+[id='proc-implementing-optimization-proposal-{context}']
+
+= Implementing an optimization proposal
+
+You can implement an xref:con-optimization-proposals-{context}[optimization proposal] if its status is `ProposalReady`.
+Cruise Control will then reassign partitions to brokers and change partition leadership, as summarized in the optimization proposal.
+
+[CAUTION]
+====
+*This is not a dry run.* Before you implement an optimization proposal, you must:
+
+* Refresh the proposal in case it has become out of date.
+* Carefully review the link:#contents-optimization-proposals[contents of the proposal].
+====
+
+.Prerequisites
+
+* You have xref:proc-generating-optimization-proposals-{context}[generated an optimization proposal].
+
+* The optimization proposal is ready to implement (the `KafkaRebalance` custom resource status is `ProposalReady`).
+
+.Procedure
+
+Perform these steps for the optimization proposal that you want to implement:
+
+. If the `KafkaRebalance` status has been `ProposalReady` for some time, it might no longer apply to the current state of the Kafka cluster. In this case, you must refresh the optimization proposal to make sure it uses the latest cluster metrics:
+
+.. Annotate the `KafkaRebalance` resource in Kubernetes:
++
+[source,shell,subs="+quotes"]
+----
+kubectl annotate kafkarebalance _cluster-rebalance-name_ strimzi.io/rebalance=refresh
+----
+
+.. Check the status of the `KafkaRebalance` resource:
++
+[source,shell,subs="+quotes"]
+----
+kubectl get kafka kafkarebalance -o jsonpath='{.status}'
```

Comment:
You should make all the "Check the status of the resource" command consistent. You uses `kubectl describe` in others?

Reply:
Yes, good point. I'll check them all.